libafl_wizard/questions.toml

```diff
@@ -0,0 +1,271 @@
+    code = """
+    /* Forkserver Executor */
+    """
+    skip = []
+
+[[question]]
+id = "corrupt memory"
+title = "Can your target corrupt memory used by the fuzzer?"
+content = """
+Under some circumstances, you may find your harness pretty unstable or your harness wreaks havoc on the global states. In this case,
+you want to fork it before executing the harness runs in the child process so that it doesn't break things.
+"""
+skipped_by = ""
+previous = ""
+
+    [[question.answers]]
+    was_chosen = false
+    answer = "Yes"
+    next = "crash/timeout"
+    code = """
+    use libafl::InProcessForkExecutor;
+    use libafl_bolts::{
+        shmem::{unix_shmem, ShMemProvider},
+        tuples::tuple_list,
+    };
+
+    let mut shmem_provider = unix_shmem::UnixShMemProvider::new().unwrap();
+
+    let mut executor = InProcessForkExecutor::new(
+        &mut harness,
+        tuple_list!(observer),
+        &mut fuzzer,
+        &mut state,
+        &mut mgr,
+        shmem_provider,
+    )
+    .expect("Failed to create the Executor");
+    """
+    skip = []
+
+    [[question.answers]]
+    was_chosen = false
+    answer = "No"
+    next = "crash/timeout"
+    code = """
+    use libafl::executors::inprocess::InProcessExecutor;
+    use libafl_bolts::tuples::tuple_list;
+
+    let mut executor = InProcessExecutor::new(
+        &mut harness,
+        tuple_list!(observer),
+        &mut fuzzer,
+        &mut state,
+        &mut mgr,
+    )
+    .expect("Failed to create the Executor");
+    """
+    skip = []
+
+[[question]]
+id = "crash/timeout"
+title = "Do you expect to cause a crash or a timeout on the target?"
+content = """
+Determining the objective of the fuzzing campaign is essential to identify input that triggered critical errors on the target.
+
+Telling the fuzzer that we are looking for a crash means that a testcase, which causes a crash on the target, fullfills the objective 
+and, differently from the ones that e.g. reach a new execution point, this testcase is saved in a separate folder for you to check the 
+input that trigerred the crash.
+
+A timeout follows the same idea: a testcase that takes longer to execute than a particular timeout.
+
+It's even possible to join these two ideas to instruct the fuzzzer that any testcase, which causes a crash or takes long enough to 
+execute, is stored.
+"""
+skipped_by = ""
+previous = ""
+
+    [[question.answers]]
+    was_chosen = false
+    answer = "Crash"
+    next = "default components"
+    code = """
+    use libafl::feedbacks::CrashFeedback;
+
+    let mut objective = CrashFeedback::new();
+    """
+    skip = []
+
+    [[question.answers]]
+    was_chosen = false
+    answer = "Crash or Timeout"
+    next = "default components"
+    code = """
+    use libafl::{
+        feedback_or_fast,
+        feedbacks::{CrashFeedback, TimeoutFeedback},
+    };
+
+    let mut objective = feedback_or_fast!(
+        CrashFeedback::new(),
+        TimeoutFeedback::new()
+    );

```

Comment:
Instead of importing I believe it would be more robust if the full-qualified path to the struct/item is used.
```rust

    let mut objective = libafl::feedback_or_fast!(
        ...
    );

```